src/components/ItemDetail/Controls.vue

```diff
@@ -13,6 +28,7 @@
 import { defineComponent } from 'vue'
 import Icon from '/@/components/UI/Icon.vue'
 import IconButton from '/@/components/UI/IconButton.vue'
+import useLending from './use/useLending'
```

Comment:
```suggestion
import useLending from './use/lending'
```
命名規則こういう風にしてほしい:pray: